backend/src/schema/resolvers/reports.js

```diff
@@ -183,4 +183,10 @@ export default {
       return reviewed
     },
   },
+  FILED: {
+    reasonDescription: async (parent, _params, _context, _resolveInfo) => {
+      if (typeof parent.reasonDescription !== 'undefined') return parent.reasonDescription
+      return null
+    },
+  },
```

Comment:
I guess we can't use the `unfefinedToNull` resolver for `relationships`??

Reply:
Thanks for the Hint! I use `undefinedToNullResolver` now …